exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_log_exporter/__init__.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         ] = None,
         timeout: Optional[float] = None,
         compression: Optional[Compression] = None,
+        channel_options: Optional[TypingSequence[Tuple[str, str]]] = None,
```

Comment:
Seems fine to me. I guess the alternative is allowing to pass a gRPC channel directly, but then the user has to configure all the other options.

Reply:
I don't expect enduser to play with this, I've opened this for being able to pass exporter params from sdk configuration https://github.com/open-telemetry/opentelemetry-python/pull/4659

Reply:
Not blocking just a callout